android/engine/src/main/java/org/smartregister/fhircore/engine/util/SharedPreferenceKey.kt

```diff
@@ -32,4 +32,5 @@ enum class SharedPreferenceKey {
   USER_INFO,
   CARE_TEAM,
   ORGANIZATION,
+  SELECTED_LOCATION_ID
```

Comment:
noted - right now it is not being used because we dont have the location selector bottom sheet but I'll sure use preference data store once this is being utilised.